File: apiome-ui/examples/pydantic/negative/06-unresolvable-import.py
"""
`shared.value_objects` is not present in the fileset, so `Money` can never be resolved.
Static analysis must report a named unresolved reference rather than modelling the field
as `Any`.
"""

from __future__ import annotations

from pydantic import BaseModel

from shared.value_objects import Money


class Invoice(BaseModel):
    invoice_id: str
    total: Money

File: apiome-ui/examples/typescript-types/negative/06-unresolvable-import.ts
// './missing-module' is not present in the fileset, so `SharedAddress` can never be
// resolved. The import must report a named unresolved reference rather than modelling
// the field as `any`.
import type { SharedAddress } from './missing-module';

export interface Shipment {
  shipmentId: string;
  destination: SharedAddress;
  weightKg: number;
}
